src/commonMain/kotlin/com/github/doyaaaaaken/kotlincsv/client/BufferedLineReader.kt

```diff
@@ -6,14 +6,20 @@ package com.github.doyaaaaaken.kotlincsv.client
 internal class BufferedLineReader(
     private val br: Reader
 ) {
+    companion object {
+        private const val BOM = '\uFEFF'
+    }
+
+    private fun StringBuilder.isEmptyLine(): Boolean =
+        this.isEmpty() || this.length == 1 && this[0] == BOM
```

Comment:
The code below seems to better grasp the intent.
`this.isEmpty() || (this.length == 1 && this[0] == BOM)`

Reply:
I prefer it too, but CodeFactor did not like it so I fixed as 5231efb3902f554cec4f9786c7c82de19af5f193 😇